pubmed_project/settings.py

```diff
@@ -86,19 +86,19 @@
 # ATTENTION !!!
 # While running in local either comment out below section or
 # use 'Production Database Configuration' with Environment Variables.
-# DATABASES = {
-#     'default': {
-#         'ENGINE': 'django.db.backends.postgresql',
-#         'NAME': 'medicles',
-#         'USER': 'postgres',
-#         'PASSWORD': 'postgres',
-#         'HOST': 'db',
-#         'PORT': '5432'
-#     }
-# }
-
-
-# Production Database Configuration with Environment Variables
+DATABASES = {
```

Comment:
This DB configuraiton should be commented out. We should use os.environ() version in production.